assets/queries/k8s/kubernetes_docker_socket_volume/query.rego

```diff
@@ -0,0 +1,19 @@
+package Cx
+
+CxPolicy [ result ] {
+	document := input.document[i]
+    metadata:= document.metadata
+    spec := document.spec
+    volumes := spec.volumes
+    volumes[c].hostPath.path == "/var/run/docker.sock"
+
+	
+
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	   sprintf("metadata.name=%s.spec.volumes.name=%s.hostPath.path", [metadata.name, volumes[c].name]),
+                "issueType":		"IncorrectValue",
+                "keyExpectedValue": sprintf("spec[%s].volumes[%s].hostPath.path is not '/var/run/docker.sock'", [metadata.name, volumes[c].name]),
+                "keyActualValue": 	sprintf("spec[%s].volumes[%s].hostPath.path is '/var/run/docker.sock'", [metadata.name, volumes[c].name])
+              }
+}
```

Comment:
This seems good if it is a pod, but we also should validate the template if it is not a pod and also check the cronjob

Reply:
And to save you some time trying to fix conflicts, I suggest testing these 3 cases separately, and not use a function common to all cases, even though all that changes is the path where the evaluated attribute is.